content/en/docs/opensubsonic-api.md

```diff
@@ -21,13 +21,13 @@ Correspondingly, there are two sets of methods for searching, starring and album
 | [Playlists](/categories/playlists)                           | [`getPlaylists`](../endpoints/getplaylists) [`getPlaylist`](../endpoints/getplaylist) [`createPlaylist`](../endpoints/createplaylist) [`updatePlaylist`](../endpoints/updateplaylist) [`deletePlaylist`](../endpoints/deleteplaylist)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| [Media retrieval](/categories/media-retrieval)               | [`stream`](../endpoints/stream) [`download`](../endpoints/download) [`hls`](../endpoints/hls) [`getCaptions`](../endpoints/getcaptions) [`getCoverArt`](../endpoints/getcoverart) [`getLyrics`](../endpoints/getlyrics) [`getAvatar`](../endpoints/getavatar)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| [Media retrieval](/categories/media-retrieval)               | [`stream`](../endpoints/stream) [`download`](../endpoints/download) [`hls`](../endpoints/hls) [`getCaptions`](../endpoints/getcaptions) [`getCoverArt`](../endpoints/getcoverart) [`getLyrics`](../endpoints/getlyrics) [`getAvatar`](../endpoints/getavatar) [getLyricsBySongId](../endpoints/getlyricsbysongid)                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
getLyricsBySongId is missing the tildes `, which makes it inconsistent with the rest of the endpoints